Template/Variable/MatomoConfigurationVariable.php

```diff
@@ -166,6 +166,11 @@ public function getParameters()
                     'Strict' => 'Strict',
                 );
             }),
+            $this->makeSetting('disableBrowserFeatureDetection', false, FieldConfig::TYPE_BOOL, function (FieldConfig $field) {
+                $field->title = Piwik::translate('TagManager_MatomoConfigurationMatomoDisableBrowserFeatureDetectionTitle');
+                $field->description = Piwik::translate('TagManager_MatomoConfigurationMatomoDisableBrowserFeatureDetectionDescription');
+                $field->inlineHelp = Piwik::translate('TagManager_MatomoConfigurationMatomoDisableBrowserFeatureDetectionInLineHelp', array('<br><strong>', '<a href="https://matomo.org/faq/how-to/how-do-i-disable-browser-feature-detection-completely/" target="_blank">', '</a>', '</strong>'));
```

Comment:
not sure why, the "target='_blank'" doesn't seem to be in the HTML for some reason. Not sure if anything is stripping this or what is happening. It opened the link in the same window. Maybe Vue is removing it? We wouldn't spend too much time on this one so we could otherwise just create a bug report in core that attributes are being ignored.

also what we good but I think no longer required is to add `rel="noreferrer noopener"` attribute.

<img width="931" alt="image" src="https://user-images.githubusercontent.com/273120/158458411-1849095a-3d20-4cbc-8ba2-4dc5f1cd7310.png">

Reply:
I had a word with dizzy on this, its due to recent vue changes, `it needs a rel="noreferrer noopener" otherwise its considered insecure` I have added the rel attribute and that seems to work